superset-frontend/src/components/AsyncAceEditor/index.tsx

```diff
@@ -167,15 +172,66 @@ export default function AsyncAceEditor(
         }, [keywords, setCompleters]);
 
         return (
-          <ReactAceEditor
-            ref={ref}
-            mode={mode}
-            theme={theme}
-            tabSize={tabSize}
-            defaultValue={defaultValue}
-            setOptions={{ fontFamily }}
-            {...props}
-          />
+          <>
+            <Global
+              styles={css`
+                .ace_tooltip {
```

Comment:
@justinpark I noticed that `src/SqlLab/components/AceEditorWrapper/index.tsx` has similar logic as this code. Is it possible to avoid code duplication in this case and also ensure similar behavior for both editors by using a single component?

Reply:
As @michael-s-molina and I discussed offline, the SqlLab/AceEditorWrapper component has styles associated with the SqlLab Editor, and this section applies the global ace-editor styles.